Week1/MAKEME.md

```diff
@@ -7,18 +7,23 @@
 
 ### 1. Testing Exercises
 
+Using **Slack** application on desktop and mobile:
+
+- Describe shared testing activities for desktop and mobile versions
+
 ### 2. Assignment
 
-Given an application with web, desktop and mobile versions (e.g. WhatsApp):
+Given [RealWorld example application](https://github.com/gothinkster/realworld) on web ([React/Redux](https://github.com/gothinkster/react-redux-realworld-example-app)) and on mobile ([NativeScript (Angular)](https://github.com/nea/nativescript-realworld-example-app))
 
-- Describe shared testing activities for desktop and mobile
-- Describe unique testing activities per platform
-- Give two possible suggestions to improve the quality of the application
+- Describe shared testing activities for web and mobile
+- Describe unique testing activities per platform (web and mobile)
+- Come up with two possible suggestions to improve the quality of the application
```

Comment:
The quality in terms of what precisely?

Reply:
That's the question :) ... they should know at this point what is quality represents and how they can improve it